file2.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE users(
+	username       TEXT PRIMARY KEY NOT NULL,
+	email          TEXT    NOT NULL,
+	password       TEXT    NULLABLE,
+	age            INT     NULLABLE,
+	admin          BOOLEAN NULLABLE,
```

Comment:
I think `admin boolean` is a reasonable choice based on what the spec called for, but I would recommend using an enum here instead. Even if it's just two values (`Admin` and `RegularUser`), it wouldn't be much more complexity but it would make it easy to extend to future use cases.

That said, I don't feel too strongly about this—it would be pretty easy to migrate to the enum later